app/frontend/courses/report/components/CoursesReport__Overview.res

```diff
@@ -213,19 +260,69 @@ let levelProgressBar = (levelId, levels, levelsCompleted) => {
+                <span> {t("milestone_status_title")->str} </span>
+                <HelpIcon className="ml-2"> {t("milestone_status_help")->str} </HelpIcon>
+              </p>
+              {milestoneTargetsCompletionStatus(overview)}
+            </div>
+            <div className="grid gap-2 mt-2">
+              {ArrayUtils.copyAndSort(
+                (a, b) =>
+                  a->CoursesReport__MilestoneTargetCompletionStatus.milestoneNumber -
+                    b->CoursesReport__MilestoneTargetCompletionStatus.milestoneNumber,
+                StudentOverview.milestoneTargetsCompletionStatus(overview),
+              )
+              ->Js.Array2.map(data => {
+                <a
+                  href={"/targets/" ++ CoursesReport__MilestoneTargetCompletionStatus.id(data)}
+                  className="flex col-span-1 items-center justify-between p-2 rounded-md border bg-gray-100 hover:bg-primary-100 hover:border-primary-500 hover:text-primary-500 transition">
+                  <div className="flex items-center">
+                    <p className="text-sm font-semibold mr-2">
+                      {("M" ++
+                      string_of_int(
+                        CoursesReport__MilestoneTargetCompletionStatus.milestoneNumber(data),
+                      ))->str}
+                    </p>
+                    <div
+                      className="flex-1 text-sm truncate max-w-[20ch] sm:max-w-[30ch] md:max-w-[28ch]">
+                      <p className="text-ellipsis overflow-hidden">
+                        {data->CoursesReport__MilestoneTargetCompletionStatus.title->str}
+                      </p>
+                    </div>
+                  </div>
+                  <div className="flex-shrink-0 me-2">
+                    <span
+                      className={"text-xs font-medium " ++ {
+                        data->CoursesReport__MilestoneTargetCompletionStatus.completed
+                          ? "text-green-700 bg-green-100 px-1 py-0.5 rounded"
+                          : "text-orange-700 bg-orange-100 px-1 py-0.5 rounded"
+                      }}>
+                      {<Icon
+                        className={data->CoursesReport__MilestoneTargetCompletionStatus.completed
+                          ? "if i-check-circle-solid text-green-600"
+                          : "if i-dashed-circle-light text-orange-600"}
+                      />}
+                      {data->CoursesReport__MilestoneTargetCompletionStatus.completed
+                        ? <span className="ms-1"> {t("milestone_completed") |> str} </span>
+                        : <span className="ms-1"> {t("milestone_pending") |> str} </span>}
+                    </span>
+                  </div>
+                </a>
+              })
+              ->React.array}
+            </div>
+          </div>
+          <div className="mt-8">
+            <p className="text-sm font-semibold"> {t("targets_overview") |> str} </p>
```

Comment:
@mahesh-krishnakumar Weren't all the sections below this point supposed to be at the top of the page?